helmfile/values/trivy/trivy-operator.yaml.gotmpl

```
@@ -30,6 +30,9 @@ tolerations:
 affinity:
 {{- toYaml .Values.trivy.affinity | nindent 2 }}
 
+resources:
+{{- toYaml .Values.trivy.resources | nindent 2 }}
```

I prefer this format.
```suggestion
resources: {- toYaml .Values.trivy.resources | nindent 2 }}
```

---

Thank you 🤗